contracts/Tasks.sol

```diff
@@ -0,0 +1,114 @@
+pragma solidity ^0.4.11;
+
+// Distense is a decentralized, for-profit code cooperative
+// INSECURE/DRAFT
+
+import './lib/SafeMath.sol';
+
+contract Tasks {
+  using SafeMath for uint256;
+
+  uint256 public numTasks;
+  uint256 public numContribs;
+
+  address  public votingAddress;
+
+  struct Task {
+    address createdBy;
+    string  title;
+    string  url;
+    string project;
+    string subProject;
+    bytes ipfsHashID; // longer than 32
```

Comment:
I added ID to also signify that the `ipfsHash`  is *also* the unique id of the task.  Still want to change?  `ipfsHashID` is quite a bit more explicit with only two added chars.  Could go either way but want to make the fact that the `ipfsHash` is also the id of the task readily apparent.